include/Packet.hpp

```diff
@@ -0,0 +1,27 @@
+#pragma once
+
+#include <cstdint>
+
+class Packet {
+public:
+    Packet(std::uint64_t sender_id, std::uint64_t sending_time_ns, std::uint64_t delivery_time_ns);
```

Comment:
Let us also include here information about the mount of transmitted data (in the form of number of successfully delivered bits). This info will be useful to calculate the delivery rate. Also, it is used in the BBR congestion control algorithm.